README.md

```diff
@@ -134,7 +134,8 @@ pods = client.get_pods(label_selector: 'name=redis-master,app=redis')
 ```
 
 #### Get a specific instance of an entity (by name)
-Such as: `get_service "service name"` , `get_pod "pod name"` , `get_replication_controller "rc name"`, `get_secret "secret name"`
+Such as: `get_service "service name"` , `get_pod "pod name"` , `get_replication_controller "rc name"`, `get_secret "secret name"`,
+`get_component_status "component name"`
```

Comment:
> is component status retrieval also working per namespace?

it works *only* per namespace, but as I noticed other entities do as well. so, should I add ', "namespace"' or leave it as is (to maintain consistency)?